cypress/utils/dashboards/workspace-plugin/commands.js

```diff
@@ -41,7 +41,7 @@ Cypress.Commands.add('deleteWorkspaceByName', (workspaceName) => {
   });
 });
 
-Cypress.Commands.add('createWorkspace', (workspaceName) => {
+Cypress.Commands.add('createWorkspace', (workspace) => {
```

Comment:
Perhaps we need to modify the place where calls this API.

Reply:
thanks, will update workspace_dropdown_menu.spec.js

Reply:
`workspace_dropdown_menu.spec.js` updated.